packages/core/src/domain/consolidated/filename.ts

```diff
@@ -1,10 +1,14 @@
 import { createFilePath } from "../filename";
 
 const extension = ".json";
-const consolidatedDataFilenameSuffix = `CONSOLIDATED_DATA${extension}`;
 
-export function createConsolidatedDataFilePath(cxId: string, patientId: string): string {
-  return createFilePath(cxId, patientId, consolidatedDataFilenameSuffix);
+export function createConsolidatedDataFilePath(
```

Comment:
I'd make these props if possible.

Reply:
Will leave as is for now and add as a note for the next PR